core/invoker/Dockerfile.s390x

```diff
@@ -0,0 +1,15 @@
+FROM scala
+
+RUN apt-get update \
+ && DEBIAN_FRONTEND="noninteractive" apt-get install -y docker.io \
```

Comment:
@jonpspri @csantanapr is it ok to have any docker version here?

Reply:
@sudswas  My gut says probably not.  I expect the Docker version should match the docker version being used to run OpenWhisk, which means locking in on a stable version of Docker CE to use.  So we should adjust the s390x and ppc64le Dockerfiles to do that.  Does it make sense for us to use the templating engine so we can provide the configuration variable externally?  I have lots of thoughts and not a clear direction yet.

Reply:
that's correct same docker version in invoker to match docker engine on the host.

Reply:
Yes that's OK to templating with variables